conf/gnss-sdr_BDS_B1C_byte.conf

```diff
@@ -0,0 +1,125 @@
+; This is a GNSS-SDR configuration file
+; The configuration API is described at https://gnss-sdr.org/docs/sp-blocks/
+
+; You can define your own receiver and invoke it by doing
+; gnss-sdr --config_file=my_GNSS_SDR_configuration.conf
+;
+
+[GNSS-SDR]
+
+;######### GLOBAL OPTIONS ##################
+;internal_fs_sps: Internal signal sampling frequency after the signal conditioning stage [samples per second].
+GNSS-SDR.internal_fs_sps=25000000
+
+;######### CONTROL_THREAD CONFIG ############
+ControlThread.wait_for_flowgraph=false
+
+;######### SIGNAL_SOURCE CONFIG ############
+SignalSource.implementation=File_Signal_Source
+SignalSource.filename=/home/andrewkamble88/GNSS-Metadata-Standard-master/install/GPSL1-Beidou3B1C.dat
+SignalSource.item_type=byte
+SignalSource.sampling_frequency=25000000
+SignalSource.samples=0
+SignalSource.repeat=false
+SignalSource.dump=false
+SignalSource.enable_throttle_control=false
+
+;######### SIGNAL_CONDITIONER CONFIG ############
+SignalConditioner.implementation=Signal_Conditioner
+DataTypeAdapter.implementation=Byte_To_Short
+InputFilter.implementation=Freq_Xlating_Fir_Filter
+InputFilter.input_item_type=short
+InputFilter.output_item_type=gr_complex
+InputFilter.taps_item_type=float
+InputFilter.number_of_taps=5
+InputFilter.number_of_bands=2
+InputFilter.band1_begin=0.0
+InputFilter.band1_end=0.70
+InputFilter.band2_begin=0.80
+InputFilter.band2_end=1.0
+InputFilter.ampl1_begin=1.0
+InputFilter.ampl1_end=1.0
+InputFilter.ampl2_begin=0.0
+InputFilter.ampl2_end=0.0
+InputFilter.band1_error=1.0
+InputFilter.band2_error=1.0
+InputFilter.filter_type=bandpass
+InputFilter.grid_density=16
+InputFilter.sampling_frequency=25000000
+InputFilter.IF=6250000
+;InputFilter.dump = false
+;InputFilter.dump_filename=
+Resampler.implementation=Pass_Through
+Resampler.sample_freq_in=25000000
+Resampler.sample_freq_out=25000000
+Resampler.item_type=gr_complex
+
+
+;######### CHANNELS GLOBAL CONFIG ############
+Channels_C1.count=10
+Channels.in_acquisition=1
+Channel.signal=C1
+
+Channel0.satellite = 6;
+Channel1.satellite = 8;
+Channel2.satellite = 9;
+Channel3.satellite = 13;
+Channel4.satellite = 17;
+Channel5.satellite = 1;
+Channel6.satellite = 2;
+Channel7.satellite = 3;
+Channel8.satellite = 4;
+Channel9.satellite = 5;
+
+;######### ACQUISITION GLOBAL CONFIG ############
+Acquisition_C1.implementation=BEIDOU_B1C_PCPS_Acquisition
```

Comment:
To test the blocks you could write unit test or test the code using the tracking and telemetry placeholdes